server/graphql-engine.cabal

```diff
@@ -158,7 +158,8 @@ flag ghci-load-test-with-lib
 common common-all
   ghc-options:
     -foptimal-applicative-do
-    -Wall -Wcompat -Wincomplete-record-updates -Wincomplete-uni-patterns -Wredundant-constraints
+    -Wall -Wcompat -Wincomplete-record-updates -Wincomplete-uni-patterns
+    -- -Wredundant-constraints
```

Comment:
Why did you have to remove this? Did you get extra warnings?

Reply:
Yes, got a lot of compilation errors for redundant-constrains and typed holes, which I found strange. Temporary fix to get a build.